packages/regl-worldview/src/WorldviewContext.js

```diff
@@ -227,7 +232,16 @@ export class WorldviewContext {
     this.counters.render = Date.now() - start;
   }
 
-  _debouncedPaint = debounce(this.paint, 10);
+  tick = () => {
+    cancelAnimationFrame(this._frame);
```

Comment:
Since the raf callback doesn't capture any local state, why do you need to cancel the pending frame if there is one already?

Reply:
Defensive programming in case `tick()` gets called a second time, it won't have two loops going

Reply:
I just mean `if (!this._frame) this._frame = ...`

Reply:
Clearing a pending async operation is idempotent, and also when the async callback runs or is canceled, that binding will still be truthy... I chose to just always clear the pending async operation before setting one, vs adding extra state to track if there is a pending async operation, if that's what you're asking.

Reply:
I guess I'm just not sure if it might be slightly more efficient to keep the existing request rather than making a new request, if you're calling this frequently. probably doesn't matter much.

Reply:
Yeah, in practice we should never get into that state, it's kind of like the trolley problem, if it actually happens, something much worse is probably wrong, but it's good to have a defense in case.